src/components/reusable/inlineCodeView/inlineCodeView.scss

```diff
@@ -0,0 +1,38 @@
+@use '../../../common/scss/global.scss';
+
+$default-dark-bg: var(--kd-color-background-ui-strong);
+$default-darker-bg: #272727;
+$default-dark-font-color: #FFFFFF;
+$default-light-bg: #F9F9F9;
+$default-light-font-color: #3D3C3C;
+
+:host {
+    display: inline-block;
+}
+
+code {
+    font-family: monospace;
+    line-height: 1.5;
+    word-break: break-word;
+    vertical-align: text-bottom;
+    margin: 0 4px;
+    padding: 4px 8px;
+    font-family: monospace;
+    font-size: max(calc(var(--inline-snippet-font-size, 14) * 1px), 14px);
+    border-radius: 1px;
```

Comment:
Maybe bump this to 2px? Couldn't really notice 1px.

Reply:
✅